grype/presenter/internal/test_helpers.go

```diff
@@ -390,8 +390,8 @@ func generatePackages(t *testing.T) []syftPkg.Package {
 				{
 					Attributes: cpe.Attributes{
 						Part:     "a",
-						Vendor:   "anchore",
-						Product:  "engine",
+						Vendor:   "ancho:re",
```

Comment:
I wonder if this test case could be a little clearer? having a vendor called `ancho:re` just looks weird. Maybe the vendor could be like `anchore:oss` or something so it looks plausible that there's meant to be a `:` in there? Not blocking feedback but it would make some of the test cases look a lot less weird/broken.